creator-cards/mohammed-alwedaei.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohammed Alwedaei",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, Cloud, Architecture",
+  "github": "https://github.com/Mohammed-Alwedaei",
+  "channels": [
+    {
+      "name": "Site",
+      "url": "https://www.pexelx.io/"
```

Comment:
It says. This site is under maintenance.

Reply:
Yes it is under maintenance, can I just remove it for now and add it back later?